base-theme/layouts/partials/site_root_url.html

```diff
@@ -1,6 +1,10 @@
-{{- $baseUrl := urls.Parse site.BaseURL -}}
-{{- if or (eq $baseUrl.Scheme nil) (eq $baseUrl.Host nil) -}}
+{{- if not (eq site.BaseURL "") -}}
```

Comment:
Can we use [errorf](https://gohugo.io/functions/errorf/) to error if `baseURL` is missing or empty?

Reply:
@noisecapella An empty `baseURL` is actually acceptable, in that case it's basically assumed that it is `/`.  This check is in place because `urls.Parse` throws a hard error if it's passed a string that it cannot disassemble.